File: data/python/DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING_MAIN.py
if __name__ == '__main__':
    param = [
    ('ydYdV',),
    ('4446057',),
    ('0111',),
    ('keEj',),
    ('642861576557',),
    ('11111000101',),
    ('ram',),
    ('09773261',),
    ('1',),
    ('AVBEKClFdj',)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        try:
            if f_filled(*parameters_set) == f_gold(*parameters_set):
                print("DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING," + sys.argv[1] + "," + str(i) + ",SUCCESS")
            else:
                print("DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING," + sys.argv[1] + "," + str(i) + ",FAILURE")
        except:
            print("DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<String> param0 = new ArrayList<>();
    param0.add("ydYdV");
    param0.add("4446057");
    param0.add("0111");
    param0.add("keEj");
    param0.add("642861576557");
    param0.add("11111000101");
    param0.add("ram");
    param0.add("09773261");
    param0.add("1");
    param0.add("AVBEKClFdj");
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("DYNAMIC_PROGRAMMING_SET_17_PALINDROME_PARTITIONING," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
